### DISCUSSION TITLE
Can it be top-level injected like bot toast, so you don't need to provide context where you use it

### DISCUSSION THREAD

**Discussion by laterdayi** [payam-zahedi/toastification#62] [Ideas]
toastification.show(
  title: 'Hello, world!'
);

**Comment by payam-zahedi** [comment]
I appreciate your Suggestion. I will carefully consider the proposed change and its impact on Toastification's API. Ensuring a clean and efficient API is a priority for me. If I find the suggested modification aligns with our goals, I will make the necessary adjustments. However, I want to avoid rushing into decisions that might lead to a messy API. Your patience is appreciated, and I will keep you informed of any progress or decisions made. Thank you for your understanding.

**Reply by laterdayi** [reply]
This is an excellent library, but every time you need to pass the context is a little inconvenient, thank you for your improvement, hope to implement a api that does not require context, very much looking forward to it

**Comment by laterdayi** [comment]
Is there any progress? Expect context not to be used
@payam-zahedi
